templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig

```diff
@@ -907,7 +907,8 @@
                                                 "img",
                                                 "form.button.upload.file",
                                                 1,
-                                                true
+                                                true,
+                                                required: 'true'
```

Comment:
Shouldn't it also work to just pass `'true'` (instead of `required: 'true'`)? We don't normally pass the parameter key

Reply:
yes, I thought it should work that way, but I do not really understand how it knows that I mean true specifically for the required attribute. Is it based on the order of the attributes?

Reply:
ok, it works this way --> https://github.com/demos-europe/demosplan-core/pull/4195/commits/2683cbe272b9c2415dcb7d11b32a4f67e5b67888